docs/pages/resources.sk.mdx

```diff
@@ -0,0 +1,30 @@
+import { Card, Cards } from 'nextra/components'
+
+# Prehľad zdrojov
+
+<Cards>
+  <Card
+    title="Knihy (books)"
```

Comment:
I am not quite sure if the english translations are needed here.

Reply:
I mean it's not bad to have, but what do you think, is it needed to understand it?